yang/frr-isisd.yang

```diff
@@ -104,6 +104,16 @@ module frr-isisd {
         description
           "This enum indicates capability for both levels.";
       }
+        enum "level2_to_level1" {
```

Comment:
As in my previous review - there are no such levels in IS-IS and this change is wrong and unnecessary. That's exactly why we have two leafs - `level` and `level-from`.